Library/Homebrew/manpages/brew.1.md

```diff
@@ -333,6 +342,9 @@ Note that these flags should only appear after a command.
     in a Xcode-only configuration since it adds tools like make to your PATH
     which otherwise build-systems would not find.
 
+  * `switch` <name> <version>:
+    Symlink keg <name> with specific <version>.
```

Comment:
Updated to `Symlink all of the specific <version> of keg <name>'s install to Homebrew prefix.`